amtl/am-hashmap.h

```diff
@@ -135,7 +140,8 @@ class HashMap : private AllocPolicy
   }
   template <typename UK>
   bool add(Insert &i, UK &&key) {
-    Entry entry(ke::Forward<UK>(key), V());
+    V value;
+    Entry entry(ke::Forward<UK>(key), ke::Move(value));
```

Comment:
As I remember, this was one of your first attempts to fix this. It might not be needed at all, so I could just remove the change if you want.